### PR TITLE
Include links to the PHP SDK

### PR DESCRIPTION
**Pull request by GrahamCampbell**
cc @jlaswell

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -22,7 +22,7 @@ icon = "fas fa-code"
...
-text = "No common event format means no common libraries, tooling, and infrastructure for delivering event data across environments. CloudEvents provides SDKs for [Go](https://github.com/cloudevents/sdk-go), [JavaScript](https://github.com/cloudevents/sdk-javascript), [Java](https://github.com/cloudevents/sdk-java), [C#](https://github.com/cloudevents/sdk-csharp), [Ruby](https://github.com/cloudevents/sdk-ruby), and [Python](https://github.com/cloudevents/sdk-python) that can be used to build event routers, tracing systems, and other tools."
+text = "No common event format means no common libraries, tooling, and infrastructure for delivering event data across environments. CloudEvents provides SDKs for [Go](https://github.com/cloudevents/sdk-go), [JavaScript](https://github.com/cloudevents/sdk-javascript), [Java](https://github.com/cloudevents/sdk-java), [C#](https://github.com/cloudevents/sdk-csharp), [Ruby](https://github.com/cloudevents/sdk-ruby), [PHP](https://github.com/cloudevents/sdk-php), [PHP](https://github.com/cloudevents/sdk-php), and [Python](https://github.com/cloudevents/sdk-python) that can be used to build event routers, tracing systems, and other tools."
```
Comment (duglin) [inline config.toml:25]: I think you listed PHO twice

Comment (GrahamCampbell) [reply config.toml:25]: Oops. Not sure how I managed that. 😅

Comment (GrahamCampbell) [reply config.toml]: Fixed.

**Comment by duglin** [comment]
cool - thanks!
LGTM

**Pull request merged**